build.sh

```diff
@@ -91,7 +90,9 @@ fi
 if hasArg --allgpuarch; then
     BUILD_ALL_GPU_ARCH=1
 fi
-if hasArg --multigpu; then
+if ahasArg --singlegpu; then
+    MULTIGPU=
+else
     MULTIGPU=--multigpu
 fi
 if hasArg deep-clean || hasArg clean; then
```

Comment:
looks like the `deep-clean` arg is no longer something that needs to be checked for.